frontend/src/metabase/components/EntityMenu.jsx

```diff
@@ -25,17 +25,29 @@ class EntityMenu extends Component {
     props: Props
 
     state = {
-        open: false
+        open: false,
+        freezeMenu: false,
+        menuItemContent: null
```

Comment:
Might be nice to add a comment here about what `freezeMenu` and `menuItemContent` are used for. For component state that's not as simple as "open" I've found that useful.